changelogs/fragments/add_vrfs_module.yaml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - Adds a new module `iosxr_vrfs` to manage VRFs on Cisco IOS-XR devices.
```

Comment:
Can we please make it just one changelog entry, we need not add 3 list entries for one module that is being added.